src/it/java/com/google/checkstyle/test/chapter5naming/rule528typevariablenames/MethodTypeParameterNameTest.java

```diff
@@ -31,18 +31,22 @@
 public class MethodTypeParameterNameTest extends AbstractGoogleModuleTestSupport {
 
     private static final String MSG_KEY = "name.invalidPattern";
-    private static String format;
+    private static final String format = getFormat();
```

Comment:
Why does this have to be a field? Can we just do:
```java
        final String format = checkConfig.getProperty("format");
```
in each unit test? I would rather throw an exception as part of the unit test than when we are initializing them.

Reply:
https://github.com/checkstyle/checkstyle/blob/399f1b48b79731d65e824eb927971ef6174c3463/src/it/java/com/google/checkstyle/test/chapter5naming/rule51identifiernames/CatchParameterNameTest.java#L40

Reply:
Done.